_templates/troubleshooting.md

```diff
@@ -0,0 +1,113 @@
+---
+title: [Title]
+description: [Brief description of the How-To article]
+type: [Troubleshooting]
+teampulseid: [number]
+ticketid: [number]
+pitsid: [number] <!-- Not sure we need this, as pits item are related to teampulse anyway -->
+<!-- TO DO: What else should we have here? -->
+---
+
+# Title
+<!-- Mandatory, Character Limit: 255 -->
+<!-- Recommendation is to have <= 75 characters for a title as this can impact indexing by search engines like Google, Bing, etc. -->
+<!-- The article title should be a combination of the main problem statement and if applicable, the key environment statement. -->
+
+
+
+<!-- Environment Table -->
+<!-- TO DO:
+     We need to check how such can be developed according to our needs:
+ 	 We don't need Header roll, but rather a title: Environment.
+     E.g. - Telerik Product, Version, SDK, SDK version, Mobile OS Vendor, Mobile OS Vendor Version, Client, Client Version, App Type (Hybrid/NativeScript/Native), 3rd party integration, Browser Type, Browser Version -->
+| Service:     | [AppBuilder], [Backend Services], [AppManager] |
+| Other stuff: | [Other stuff here]                             |
+
+
+
+## Description
+<!-- Mandatory -->
+State the question.
+
+* Use complete thoughts.
+* Use words and phrases the customer used to describe the issue (even if technically inaccurate).  
+* If possible restate the problem in technical wording. This increases the ability to find the article for internal staff.
+
+## Steps to Reproduce\Additional Details
+<!-- Optional -->
+<!-- TO DO: Decide which name suits better -->
+Environment information that is not listed in the Environment section of the article.
+Clarifying information that does not go in the Problem Description or Error Message.
+
+Steps to reproduce an issue are required if:
+
+* they are easily reproduced
+* do not require customer proprietary code
+* are so complex that they will not make sense to the reader.
+
+TSEs should supply steps to reproduce the issue when available.
+
+Add factual statements that clarify the issue.  These are facts that will not change when the issue is resolved.
+Information or troubleshooting steps that bring clarity to the issue but are not part of the actual problem.
+
+## Error Message
+<!-- Optional -->
+List the exact error message(s) pertaining to the issue.
+Remove any customer-specific information from the error message, such as product names or file names.
+
+## Cause
```

Comment:
We will stick to Cause\Possible Cause\Possible Causes depending on the scenario.